manticore/platforms/evm.py

```diff
@@ -0,0 +1,1760 @@
+                0x96: ('SWAP', 0, 8, 8, 0, 'Exchange 1st and 8rd stack items.'),
+                0x97: ('SWAP', 0, 9, 9, 0, 'Exchange 1st and 9rd stack items.'),
+                0x98: ('SWAP', 0, 10, 10, 0, 'Exchange 1st and 10rd stack items.'),
+                0x99: ('SWAP', 0, 11, 11, 0, 'Exchange 1st and 11rd stack items.'),
+                0x9a: ('SWAP', 0, 12, 12, 0, 'Exchange 1st and 12rd stack items.'),
+                0x9b: ('SWAP', 0, 13, 13, 0, 'Exchange 1st and 13rd stack items.'),
+                0x9c: ('SWAP', 0, 14, 14, 0, 'Exchange 1st and 14rd stack items.'),
+                0x9d: ('SWAP', 0, 15, 15, 0, 'Exchange 1st and 15rd stack items.'),
+                0x9e: ('SWAP', 0, 16, 16, 0, 'Exchange 1st and 16rd stack items.'),
+                0x9f: ('SWAP', 0, 17, 17, 0, 'Exchange 1st and 17th stack items.'),
+                0xa0: ('LOG', 0, 2, 0, 375, 'Append log record with no topics.'),
+                0xa1: ('LOG', 0, 3, 0, 750, 'Append log record with one topic.'),
+                0xa2: ('LOG', 0, 4, 0, 1125, 'Append log record with two topics.'),
+                0xa3: ('LOG', 0, 5, 0, 1500, 'Append log record with three topics.'),
+                0xa4: ('LOG', 0, 6, 0, 1875, 'Append log record with four topics.'),
+                0xf0: ('CREATE', 0, 3, 1, 32000, 'Create a new account with associated code.'),
+                0xf1: ('CALL', 0, 7, 1, 40, 'Message-call into an account.'),
+                0xf2: ('CALLCODE', 0, 7, 1, 40, "Message-call into this account with alternative account's code."),
+                0xf3: ('RETURN', 0, 2, 0, 0, 'Halt execution returning output data.'),
+                0xf4: ('DELEGATECALL', 0, 6, 1, 40, "Message-call into this account with an alternative account's code, but persisting into this account with an alternative account's code."),
+                0xf5: ('BREAKPOINT', 0, 0, 0, 40, 'Not in yellow paper FIXME'),
+                0xf6: ('RNGSEED', 0, 1, 1, 0, 'Not in yellow paper FIXME'),
+                0xf7: ('SSIZEEXT', 0, 2, 1, 0, 'Not in yellow paper FIXME'),
+                0xf8: ('SLOADBYTES', 0, 3, 0, 0, 'Not in yellow paper FIXME'),
+                0xf9: ('SSTOREBYTES', 0, 3, 0, 0, 'Not in yellow paper FIXME'),
+                0xfa: ('SSIZE', 0, 1, 1, 40, 'Not in yellow paper FIXME'),
+                0xfb: ('STATEROOT', 0, 1, 1, 0, 'Not in yellow paper FIXME'),
+                0xfc: ('TXEXECGAS', 0, 0, 1, 0, 'Not in yellow paper FIXME'),
+                0xfd: ('REVERT', 0, 2, 0, 0, 'Stop execution and revert state changes, without consuming all provided gas and providing a reason.'),
+                0xfe: ('INVALID', 0, 0, 0, 0, 'Designated invalid instruction.'),
+                0xff: ('SELFDESTRUCT', 0, 1, 0, 0, 'Halt execution and register account for later deletion.')
+            }
+
+
+    @staticmethod
+    def decode_one(bytecode):
+        '''
+        '''
+        bytecode = iter(bytecode)
```

Comment:
This looks redundant, it's already made into an iterator in `decode_all`

Reply:
Unless it's called from other func. I can make it private if this is not acceptable.